resources/generate_book.py

```diff
@@ -659,6 +659,62 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
+            empty_last_page()
+        )),
+        entry('dairy', 'Dairy Products', 'tfc:food/cheese', pages=(
+            text('$(thing)Dairy$() is a $(l:the_world/food_and_water#nutrients)Nutrient$() obtained from the milk produced by $(l:mechanics/animal_husbandry#dairy_animals)Dairy Animals$(). It can be drank, or processed into $(thing)Cheese$(). Drinking can be done out of a jug, and always restores $(l:the_world/food_and_water#thirst)Thirst$(). However, it only adds to nutrition when drank after eating a food. Practically, this means that drinking milk twice in a row is ineffectual. A meal must precede it.', title='Dairy'),
+            text('To start the $(thing)Cheesemaking$() process, add $(thing)Milk$() and $(thing)Vinegar$() in a $(l:mechanics/barrel)Barrel$() at a ratio of 9:1. This is easiest done by filling 9 buckets of milk in a barrel, and adding a single bucket of vinegar. This produces $(thing)Milk Vinegar$().'),
+            sealed_barrel_recipe('tfc:barrel/curdling', 'Once milk and vinegar are mixed, it will curdle if it is sealed in a barrel for 8 hours. This requires no extra ingredients, except time.'),
+            sealed_barrel_recipe('tfc:barrel/cheese', 'Cheese is then made by once again sealing the curdled milk in a barrel. Cheese is a long-lasting dairy product, and can be used in some meals to add dairy to them, such as $(l:mechanics/sandwiches)Sandwiches$().')
+        )),
+        entry('scribing_table', 'Scribing Table', 'minecraft:black_dye', pages=(
+            text('The $(thing)Scribing Table$() is used to rename items. It requires $(thing)Black Dye$() to rename items, which can be supplied as the traditional dye item or as a bucket of dye fluid.'),
+            crafting('tfc:crafting/wood/birch_scribing_table', text_contents='The scribing table crafting recipe.'),
+            image('tfc:textures/gui/book/gui/scribing.png', text_contents='The scribing screen takes text entry at the top, an input on the left, a dye in the center. The output is taken from the right slot.'),
+            block_spotlight('', 'The scribing table.', 'tfc:wood/scribing_table/kapok')
+        )),
+        entry('advanced_building_materials', 'Advanced Materials', 'tfc:brick/rhyolite', pages=(
+            text('As you progress in TFC, you may want for stronger and prettier building materials, as well as some devices you may remember from vanilla. One such material is $(thing)Stone Bricks$(), which are made with $(thing)Bricks$() and $(thing)Mortar$()'),
+            instant_barrel_recipe('tfc:barrel/limewater', 'To make mortar, $(thing)Limewater$() must first be made with $(thing)Water$() and $(l:mechanics/flux)Flux$() in a $(l:mechanics/barrel)Barrel$().'),
```

Comment:
Refer to `tfc:mechanics/barrels#limewater` rather than including the recipe here? (remove a page)